api/organization/get-org-by-name.js

```diff
@@ -0,0 +1,33 @@
+const { MSGS: { INTERNAL_SERVER_ERROR }, CODE_HOSTS } = require('../../helpers/constants')
+
+module.exports = async (req, res, ctx) => {
+  try {
+    const { name } = req.query
+    ctx.log.info('finding org with name %s', name)
+
+    if (!name) {
+      res.status(404)
+      return res.send({ success: false })
+    }
+
+    let org = await ctx.db.organization.getByNameAndHost({ name, host: CODE_HOSTS.GitHub })
```

Comment:
i think the route should be explicitly github-specific OR pass in the host in the body